src/main/java/homework3/Task5.java

```diff
@@ -0,0 +1,13 @@
+package homework3;
+
+public class Task5 {
+    public static void main(String[] args) {
+        double Celsium = 5;
+        double Fahrenheit = Celsium * (9/5) + 32;
```

Comment:
Fahrenheit calculation is incorrect
5 C -> 41 F
In your case it is 37. Think why